src/wrappers/themis/Obj-C/objcthemis/scell_context_imprint.h

```diff
@@ -47,28 +47,91 @@ NS_ASSUME_NONNULL_BEGIN
 @interface TSCellContextImprint : TSCell
 
 /**
-* @brief Initialize Secure cell object in context imprint mode
-* @param [in] key master key
-*/
+ * Initialise Secure Cell in Context Imprint mode with a master key.
+ *
+ * @param [in] key  non-empty master key
+ *
+ * @returns @c nil if key is empty.
+ */
 - (nullable instancetype)initWithKey:(NSData *)key;
 
 /**
-* @brief Wrap message with context
-* @param [in] message message to wrap
-* @param [in] context user context
-* @param [in] error pointer to Error on failure
-* @return Wrapped message as NSData object on success or nil on failure
-*/
-- (nullable NSData *)wrapData:(NSData *)message context:(NSData *)context error:(NSError **)error;
+ * Encrypt data.
+ *
+ * @param [in]  message data to encrypt, must not be empty
+ * @param [in]  context associated context, must not be empty
+ * @param [out] error   optional error output
+ *
+ * Data is encrypted and context is cryptographically mixed with the data, but not included
+ * into the resulting encrypted message. You will have to provide the same context again
+ * during decryption. Usually this is some plaintext data associated with encrypted data,
+ * such as database row number, protocol message ID, etc.
```

Comment:
```suggestion
 * Data is encrypted and context is cryptographically mixed with the data, but not included
 * into the resulting encrypted message. You will have to provide the same context again
 * during decryption. Usually this is some plaintext data associated with encrypted data,
 * such as database row number, protocol message ID, etc.
 * Length preserving encryption mode without enforced integrity check.
 * @link https://docs.cossacklabs.com/pages/secure-cell-cryptosystem/#context-imprint-mode
```

Reply:
I believe this better be a part of class description rather than encryption function call. It's more likely that people will type `TSCell`, see four possible construction options, and then go off reading their descriptions.

https://github.com/cossacklabs/themis/blob/83f25359a300aa7b9e91b75d8f35b5304c2e659d/src/wrappers/themis/Obj-C/objcthemis/scell_context_imprint.h#L32-L43

I did not update them for some reason, but now that you've mentioned it, it's probably a good time to do it.